### PR TITLE
[Docs] Fix typo in docs of custom event handler

### PR DESCRIPTION
**Pull request by bowenliang123**
### _Why are the changes needed?_

- to fix typo in dependency reference in doc of custom event handler

image

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/extensions/server/events.rst [hunk 1]
```diff
@@ -55,8 +55,8 @@ To create custom EventHandlerProvider class derived from the above interface, we
 
    <dependency>
       <groupId>org.apache.kyuubi</groupId>
-      <artifactId>kyuubi-event_2.12</artifactId>
-      <version>1.7.0-incubating</version>
+      <artifactId>kyuubi-events_2.12</artifactId>
+      <version>1.7.0</version>
```
Comment (yaooqinn) [inline docs/extensions/server/events.rst:59]: You can use the directive - ` release`.

Comment (bowenliang123) [reply docs/extensions/server/events.rst:59]: Tried to use the directive， but it is not working (in code block).

```
<version>| release |</version>

<version>|release|</version>

<version>\ | release \ |</version>
```

Comment (yaooqinn) [reply docs/extensions/server/events.rst:59]: It works.

Comment (bowenliang123) [reply docs/extensions/server/events.rst]: OK, addressed. Since the directive requries `parsed-literal`, replace the `code-block` to `parsed-literal`

**Review by yaooqinn — APPROVED** [review]

File: docs/extensions/server/events.rst [hunk 2]
```diff
@@ -51,12 +51,12 @@ To create custom EventHandlerProvider class derived from the above interface, we
 
 - Referencing the library
 
-.. code-block:: xml
+.. parsed-literal::
 
    <dependency>
       <groupId>org.apache.kyuubi</groupId>
       <artifactId>kyuubi-event_2.12</artifactId>
```
Comment (yaooqinn) [inline docs/extensions/server/events.rst:58]:
```suggestion
      <artifactId>kyuubi-events_2.12</artifactId>
```

Comment (bowenliang123) [reply docs/extensions/server/events.rst]: Thanks. Updated the screenshot as well.

**Review by cxzl25 — APPROVED** [review]

**Comment by bowenliang123** [comment]
Thanks, merged to master.

**Comment by pan3793** [comment]
cherry-picked to 1.7 since it's a fix

**Comment by bowenliang123** [comment]
> cherry-picked to 1.7 since it's a fix

Not sure whether the directive is available in branch-1.7 ?

**Pull request closed without merging**